src/main/java/car/domain/CarName.java

```diff
@@ -0,0 +1,51 @@
+package car.domain;
+
+
+import java.util.Objects;
+
+public class CarName {
+    private static final int NAME_LIMIT_LENGTH = 5;
+    private static final String OVER_NAME_ERROR_MESSAGE = "이름은 %d글자 초과일 수 없습니다.";
+
+    private final String name;
+
+    private CarName(final String name) {
+        this.name = name;
+        validateNameLength(name);
+    }
```

Comment:
```suggestion
    private CarName(final String name) {
        validateNameLength(name);
        this.name = name;
    }
```
유효성 검사를 먼저하는 것이 어떨까요?

Reply:
네..! 유효성 검사하는 측면이 조금 더 좋은 코드인거 같네요